app/controllers/search_controller.rb

```diff
@@ -1,8 +1,19 @@
 class SearchController < ApplicationController
   def index
     search = params[:search].split(":").first
-    @repositories = policy_scope(Repository).search(search)
-    @teams = policy_scope(Team).search(search)
-    @namespaces = policy_scope(Namespace).search(search)
+    @search_query = params[:search]
+    @search_type = params[:type]
```

Comment:
With `params.require(:type)` the search broke when the `search` parameter is empty :(